src/store/level-wrapper.ts

```diff
@@ -205,6 +205,16 @@ export class LevelWrapper<V> {
     return executeUnlessAborted(this.db.batch(operations), options?.signal);
   }
 
+  /**
+   * @returns a LevelWrapperBatchOperation for the specified partition name
+   */
+  partitionOperation(name: string, operation: LevelWrapperBatchOperation<V>): LevelWrapperBatchOperation<V> {
```

Comment:
Nit: start methods with a verb:

```suggestion
  createPartitionOperation(partitionName: string, operation: LevelWrapperBatchOperation<V>): LevelWrapperBatchOperation<V> {
```